packages/shared/package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "@alephium/shared",
   "version": "0.7.6",
-  "main": "lib/index.ts",
+  "private": true,
+  "main": "dist/index.js",
   "browser": "dist/alephium.min.js",
-  "types": "lib/index.ts",
+  "types": "dist/index.d.ts",
   "exports": {
     ".": {
-      "default": "./lib/index.ts",
+      "default": "./dist/index.js",
```

Comment:
It was a pain in the 🍑 to figure out polyfills, so I just reverted to exactly the same config we had before with the SDK, when everything was working fine. As discussed, we need to review the code in the shared package and strip away things that might not be necessary, review our dependencies, etc.